Security/LoginManager.php

```diff
@@ -48,21 +48,29 @@ class LoginManager implements LoginManagerInterface
     private $container;
 
     /**
+     * @var RememberMeServicesInterface
+     */
+    private $rememberMeService;
+
+    /**
      * LoginManager constructor.
      *
      * @param TokenStorageInterface                  $tokenStorage
      * @param UserCheckerInterface                   $userChecker
      * @param SessionAuthenticationStrategyInterface $sessionStrategy
      * @param ContainerInterface                     $container
+     * @param RememberMeServicesInterface|null       $rememberMeService
      */
     public function __construct(TokenStorageInterface $tokenStorage, UserCheckerInterface $userChecker,
                                 SessionAuthenticationStrategyInterface $sessionStrategy,
-                                ContainerInterface $container)
-    {
+                                ContainerInterface $container,
+                                RememberMeServicesInterface $rememberMeService = null
+    ) {
         $this->tokenStorage = $tokenStorage;
         $this->userChecker = $userChecker;
         $this->sessionStrategy = $sessionStrategy;
         $this->container = $container;
```

Comment:
the container is not needed anymore IMO. We should inject the RequestStack directly instead